test/support.ts

```diff
@@ -164,7 +164,7 @@ export function createSequelizeInstance(options: Options = {}): Sequelize {
     dialectOptions: options.dialectOptions || config.dialectOptions || {},
     minifyAliases: options.minifyAliases || config.minifyAliases,
     // the test suite was written before ALS was turned on by default.
```

Comment:
```suggestion
    // the test suite was written before CLS was turned on by default.
```
Or AsyncLocalStorage

Reply:
Done